### ISSUE TITLE
Extension doesn't work in 0.9.0

### ISSUE THREAD

**Issue by jonsa** [elm-tooling/elm-language-client-vscode#80]
Something seems to have broken in 0.9.0. When I install that version nothing Elm related like syntax highlighting seems to work. If I revert back to 0.8.0 it's working again.

## Your Environment
Version: 1.42.1 (user setup)
Commit: c47d83b293181d9be64f27ff093689e8e7aed054
Date: 2020-02-11T14:45:59.656Z
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Elm extension: 0.9.0

**Comment by Razzeee** [comment]
Is there anything in the debug log of elmLS?

**Comment by jonsa** [comment]
As in #81 I don't get anything in output from extensions.

What I do notice, don't know if it's relevant thou, is that the `server` folder of the extensions seems to be missing files on `0.9.0`

`0.9.0`
![image](https://user-images.githubusercontent.com/807381/74727739-e82f7e80-5241-11ea-8a6a-55da9b2ccf7a.png)

`0.8.0`
![image](https://user-images.githubusercontent.com/807381/74727802-fd0c1200-5241-11ea-9c5e-ff45a6b9d2f1.png)

Where `out` is containing some `js` files.

**Comment by jonsa** [comment]
I do get something if I look in the right place :)

`Log (Extension host)`
```
[2020-02-18 11:20:28.557] [exthost] [error] Activating extension elmTooling.elm-ls-vscode failed due to an error:
[2020-02-18 11:20:28.560] [exthost] [error] Error: Cannot find module 'request'
```

`Log (Window)`
```
[2020-02-18 08:17:32.215] [renderer3] [error] Unable to load and parse grammar for scope source.elm from file:///c%3A/.../.vscode/extensions/elmtooling.elm-ls-vscode-0.9.0/syntaxes/elm.json {"fileOperationResult":1,"options":{"preferUnbuffered":true}}
[2020-02-18 08:17:32.216] [renderer3] [error] No grammar provided for <source.elm>: Error: No grammar provided for <source.elm>
    at Registry._collectDependenciesForDep (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3722:23)
    at Registry.<anonymous> (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3762:34)
    at step (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3636:23)
    at Object.next (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3617:53)
    at fulfilled (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3608:58)
[2020-02-18 08:18:31.168] [renderer3] [error] Unable to load and parse grammar for scope source.elm from file:///c%3A/.../.vscode/extensions/elmtooling.elm-ls-vscode-0.9.0/syntaxes/elm.json {"fileOperationResult":1,"options":{"preferUnbuffered":true}}
[2020-02-18 08:18:31.170] [renderer3] [error] No grammar provided for <source.elm>: Error: No grammar provided for <source.elm>
    at Registry._collectDependenciesForDep (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3722:23)
    at Registry.<anonymous> (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3762:34)
    at step (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3636:23)
    at Object.next (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3617:53)
    at fulfilled (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3608:58)
[2020-02-18 08:21:05.749] [renderer3] [error] Unable to load and parse grammar for scope source.elm from file:///c%3A/.../.vscode/extensions/elmtooling.elm-ls-vscode-0.9.0/syntaxes/elm.json {"fileOperationResult":1,"options":{"preferUnbuffered":true}}
[2020-02-18 08:21:05.750] [renderer3] [error] No grammar provided for <source.elm>: Error: No grammar provided for <source.elm>
    at Registry._collectDependenciesForDep (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3722:23)
    at Registry.<anonymous> (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3762:34)
    at step (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3636:23)
    at Object.next (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3617:53)
    at fulfilled (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3608:58)
[2020-02-18 11:20:27.367] [renderer3] [error] Unable to load and parse grammar for scope source.elm from file:///c%3A/.../.vscode/extensions/elmtooling.elm-ls-vscode-0.9.0/syntaxes/elm.json {"fileOperationResult":1,"options":{"preferUnbuffered":true}}
[2020-02-18 11:20:27.368] [renderer3] [error] No grammar provided for <source.elm>: Error: No grammar provided for <source.elm>
    at Registry._collectDependenciesForDep (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3722:23)
    at Registry.<anonymous> (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3762:34)
    at step (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3636:23)
    at Object.next (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3617:53)
    at fulfilled (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3608:58)
[2020-02-18 11:37:27.788] [renderer3] [error] Unable to load and parse grammar for scope source.elm from file:///c%3A/.../.vscode/extensions/elmtooling.elm-ls-vscode-0.9.0/syntaxes/elm.json {"fileOperationResult":1,"options":{"preferUnbuffered":true}}
[2020-02-18 11:37:27.789] [renderer3] [error] No grammar provided for <source.elm>: Error: No grammar provided for <source.elm>
    at Registry._collectDependenciesForDep (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3722:23)
    at Registry.<anonymous> (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3762:34)
    at step (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3636:23)
    at Object.next (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3617:53)
    at fulfilled (...\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate\release\main.js:3608:58)
```

**Comment by Razzeee** [comment]
I guess this might help? https://github.com/elm-tooling/elm-language-server/commit/bea293f0ab99e43a2f4784d0ace97403529c7e23

**Comment by jonsa** [comment]
I don't know if it was a bad upgrade from `0.8.0` or something. I uninstalled the extension from VSCode and removed all the version of the extension in `.vscode\extensions`. When I reinstalled the extension it started working.

The culprit seems to have been that `npm install` had not been executed in the extension folder, because I noticed that there was no `node_modules` folder in it. So I don't think the commit you did to the language server is necessary. A clean install of the extension fixed the issue for me.
